FLAnimatedImageDemo/FLAnimatedImage/FLAnimatedImageView.h

```diff
@@ -24,6 +24,7 @@
 // Setting `[UIImageView.image]` to a non-`nil` value clears out existing `animatedImage`.
 // And vice versa, setting `animatedImage` will initially populate the `[UIImageView.image]` to its `posterImage` and then start animating and hold `currentFrame`.
 @property (nonatomic, strong) FLAnimatedImage *animatedImage;
+@property (nonatomic, copy) void(^loopCompletionBlock)(FLAnimatedImageView *imageView, NSUInteger loopLeft);
```

Comment:
What is the first parameter -- the `imageView` itself -- used for? Is that a common convention, like in a delegate protocol, to make the first parameter return `self`?